app/src/main/java/com/eveningoutpost/dexdrip/Models/Tomato.java

```diff
@@ -212,12 +213,6 @@ static void InitBuffer(int expectedSize) {
 
         s_state = TOMATO_STATES.REQUEST_DATA_SENT;
 
-        // For debug, make it send data every minute (ERROR - We fail to send this message... needs more work,
-        // not sure it works at all)
-        ByteBuffer newFreqMessage = ByteBuffer.allocate(2);
-        newFreqMessage.put(0, (byte) 0xD1);
-        newFreqMessage.put(1, (byte) 0x05);
-        ret.add(newFreqMessage);
```

Comment:
What is the effect of removing this? I thought this enabled the required scheduler. This seems unrelated to adding Wifi support?

Reply:
Well, from the tomato standpoint, this code is not needed.
When it comes from the factory, it sends data every 5 minutes. Also when we start a new code we tell it to send every 5 minutes.

That said, since code now sends each buffer in a 150 ms period, this has influence on different androids stack implementations (actually if only one buffer is being sent, the wait will not happen), so I'll revert it and we can think if we are brave enough to change it.